source/agora/consensus/Ledger.d

```diff
@@ -1594,6 +1596,29 @@ unittest
     auto average_tx = genesisSpendable().front().refund(WK.Keys[0].address).deduct(10.coins).sign();
     assert(ledger.acceptTransaction(average_tx, 0, min_fee_pct) is null);
 
+    // switch to a different input, even with low fees this should be accepted since pool is almost empty
```

Comment:
All the added lines in the bracket should be indented.

Reply:
They result in a hard to follow diff tho

Reply:
Indentation is the basic requirement, I guess. But it's not the critical one.